addons/web/static/src/views/fields/float_time/float_time_field.xml

```diff
@@ -3,7 +3,7 @@
 
     <t t-name="web.FloatTimeField" owl="1">
         <span t-if="props.readonly" t-esc="formattedValue" />
-        <input t-else="" t-att-id="props.id" t-ref="numpadDecimal" t-att-placeholder="props.placeholder" inputmode="numeric" class="o_input" />
+        <input t-else="" type="text" t-att-id="props.id" t-ref="numpadDecimal" t-att-placeholder="props.placeholder" inputmode="numeric" class="o_input" />
```

Comment:
Is this change really needed?

Reply:
Hello @caburj , in fact it was to avoid a border when the user selects the input to edit it. 🙂

Reply:
Shoot. I should carefully judge even this very subtle change.

Reply:
I did the change in #115736 🙂